packages/eas-cli/src/commands/workflow/create.ts

```diff
@@ -0,0 +1,107 @@
+import chalk from 'chalk';
+import fs from 'fs/promises';
+import path from 'path';
+import prompts from 'prompts';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import Log from '../../log';
+
+const WORKFLOWS_DIR = '.eas/workflows';
```

Comment:
Do we want to make it work on Windows as well? Our general policy is that Windows is not officially supported and we recommend running things through WSL on Windows, but when possible we try to make it work (although we don't test against it)

I'm not sure if it will work on Windows, I believe it's safer to do:

```suggestion
const WORKFLOWS_DIR = path.join('.eas', 'workflows');
```